UefiPayloadPkg/SPI/SPI_fvb.c

```diff
@@ -0,0 +1,42 @@
+/** @file  BlSMMStoreDxe.c
+
+  Copyright (c) 2020, 9elements Agency GmbH<BR>
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Include/Library/DebugLib.h>
+#include <Library/UefiBootServicesTableLib.h>
+#include <Protocol/FirmwareVolumeBlock.h>
+#include "fch_spi_ctrl.h"
+#include "Fvb.h"
+
+EFI_FIRMWARE_VOLUME_BLOCK2_PROTOCOL FvbProtocol = {
+  NULL,             // GetAttributes
```

Comment:
These also must be filled:
- GetAttributes
- SetAttributes
- GetPhysicalAddress